command_set.go

```diff
@@ -236,6 +236,36 @@ func (cs *CommandSet) DeriveKey(path string) error {
 	return cs.checkOK(resp, err)
 }
 
+func (cs *CommandSet) ExportKey(derive bool, makeCurrent bool, onlyPublic bool, path string) ([]byte, error) {
+	var p1 uint8
+	if (derive == false) {
+		p1 = 0x00
```

Comment:
can we create some constants for these values so that they are used in `NewCommandExportKey` too?